features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/async/RecordedDataQueueHandler.kt

```diff
@@ -83,6 +83,12 @@ internal class RecordedDataQueueHandler {
         this.recordedDataQueue = recordedQueue
     }
 
+    @Synchronized
+    internal fun clearAndStopProcessingQueue() {
+        recordedDataQueue.clear()
+        executorService.shutdown()
```

Comment:
should it be `shutdownNow`? Not sure if will make the difference though in this particular case, but `shutdown` will wait until already started tasks are completed.

Reply:
yes....that's what I want, I want to avoid stopping anything in progress as it can create more issues.